lib/haml/filters/growstuff_markdown.rb

```diff
@@ -2,55 +2,69 @@
 
 module Haml::Filters
   module GrowstuffMarkdown
+    include Haml::Filters::Base
+
+    def render(text)
+      @expanded = text
```

Comment:
I kinda preferred passing `expanded` in and out of functions rather than updating an instance variable, but that's because (a) I spend too much time hanging out with functional programmers, (b) I've been bitten in the past by code with really horrible convoluted dataflow. So I guess I'm OK with this.

Reply:
I did that first -- but it didn't read as clean -- because everything called looks like

`
expanded = expand_crops(expanded).
`
that looks less like English.

Reply:
There might be another way though. This isn't the last re-factor of this module i think